packages/react-components/react-dialog/library/src/components/Dialog/Dialog.types.ts

```diff
@@ -96,8 +96,8 @@ export type DialogProps = ComponentProps<Partial<DialogSlots>> & {
   // eslint-disable-next-line @nx/workspace-consistent-callback-type -- can't change type of existing callback
   onOpenChange?: DialogOpenChangeEventHandler;
   /**
-   * Can contain two children including {@link DialogTrigger} and {@link DialogSurface}.
-   * Alternatively can only contain {@link DialogSurface} if using trigger outside dialog, or controlling state.
+   * Can contain two children including DialogTrigger and DialogSurface.
+   * Alternatively can only contain DialogSurface if using trigger outside dialog, or controlling state.
```

Comment:
_nit:_ You could in its place use backticks to make the text look different

```suggestion
   * Can contain two children including `DialogTrigger` and `DialogSurface`.
   * Alternatively can only contain `DialogSurface` if using trigger outside dialog, or controlling state.
```